M/Madagascar/build_tarballs.jl

```diff
@@ -3,37 +3,54 @@
 using BinaryBuilder, Pkg
 
 name = "Madagascar"
-version = v"1.0.0"
+version = v"1.0.1"
 
 # Collection of sources required to complete build
 sources = [
-    GitSource("https://github.com/ahay/src.git", "3a10302c63c69317df3f6f741347e7f77a2f3d58")
+    GitSource("https://github.com/ahay/src.git", "3a10302c63c69317df3f6f741347e7f77a2f3d58"),
+    DirectorySource("./bundled")
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
-cd $WORKSPACE/srcdir/src
+# Add host gcc for makefont
+apk add gcc libc-dev
 
-# install_license
-install_license COPYING.txt
+cd $WORKSPACE/srcdir/
 
-./configure --prefix=${prefix} CC=$(which gcc) CXX=$(which g++) CAIROPATH=${includedir}/cairo FFMPEGPATH=${includedir}/libavcodec/ AR=$(which ar) CPPPATH=${includedir} LIBPATH=${libdir} BLAS=openblas
+for f in ${WORKSPACE}/srcdir/patches/*.patch; do
+    atomic_patch -p1 ${f}
+done
 
-make -j ${nproc}
+cd src
+
+export SCONSFLAGS="-j ${nproc}"
+
+./configure --prefix=${prefix} CC=$CC CXX=$CXX \
+            FFMPEGPATH=${includedir}/libavcodec/ \
+            CAIROPATH=${includedir}/cairo \
+            CPPPATH=${includedir} \
+            LIBPATH=${libdir} \
+            BLAS=blastrampoline
+
+make
 make install
 """
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
-platforms = [
-    Platform("i686", "linux"; libc = "glibc"),
-    Platform("x86_64", "linux"; libc = "glibc")
-]
+#platforms = [
+#    Platform("i686", "linux"; libc = "glibc"),
+#    Platform("x86_64", "linux"; libc = "glibc"),
+#    Platform("x86_64", "linux"; libc = "musl"),
+#    Platform("i686", "linux"; libc = "musl")
+#]
+platforms = expand_cxxstring_abis(supported_platforms(;experimental=true))
```

Comment:
`experimental` is ignored, it's just kept for compatibility
```suggestion
platforms = expand_cxxstring_abis(supported_platforms())
```